about.html

```diff
@@ -0,0 +1,307 @@
+            <div class="col-8">
+                <ul>
+                    <li>
+                        <a href="index.html">Home</a>
+                    </li>
+                    <li>
+                        <a href="##">Program</a>
+                    </li>
+                    <li>
+                        <a href="##">Join</a>
+                    </li>
+                    <li>
+                        <a href="##">Sponsor</a>
+                    </li>
+                    <li>
+                        <a href="##">News</a>
+                    </li>
+                    <li>
+                        <a href="##">Follow us</a>
+                    </li>
+                </ul>
+            </div>
+
+        </div>
+
+
+        <!--main section-->
+        <div class="container-fluid main-section">
+             
+
+            <div class="container hambuger" id="hambuger_about">
+                <div class="row">
+                    <div class="col-1 hambuger-menu">
+                        <img src="./images/images/mobile_menu.png" alt="menu-bar">
+
+                    </div>
+                </div>
+
+                <!--Hamburger drop down-->
+                <div class="container-fluid hide-menu" id="hide_menu_about">
+                    <div class="row">
+                        <div class="col-12 drop-down-menu">
+                            
+                            <ul>
+                                <li>
+                                    <a href="index.html">Home</a>
+                                </li>
+                                <li>
+                                    <a href="##">Program</a>
+                                </li>
+                                <li>
+                                    <a href="##">Join</a>
+                                </li>
+                                <li>
+                                    <a href="##">Sponsor</a>
+                                </li>
+                                <li>
+                                    <a href="##">News</a>
+                                </li>
+                                <li>
+                                    <a href="##">Follow us</a>
+                                </li>
+                            </ul>
+                            <span class="closeButton" id="close_button_about">&times;</span>
+                        </div>
+                    </div>
+                </div>
+
+
+            </div>
+            <div class="row">
+
+                <div class="container">
+                    <div class="row">
+                        <div class="col-3 desk-div"></div>
+                        <div class="col-9">
+                            <h1 class="about_header">Agricultural Conference
+                                <br> West Africa Youths Submit 2023
+                            </h1>
+
+                        </div>
+
+                    </div>
+
+
+                    <div class="row">
```

Comment:
- [ ] I noticed that this part is different from the original design.  Please use the same spacing padding/margin, width, height, and color, and make sure that you use the correct font weight, font size for your text and  your different elements to make them similar to the original design and give a better user experience :+1: 

#### Your design
![Screenshot_20230113_115108](https://user-images.githubusercontent.com/6615394/212317159-88964d94-60ad-4401-8279-8d0fa0a4da6a.jpg)

#### The original design
![Screenshot_20230113_121002(1)](https://user-images.githubusercontent.com/6615394/212317575-56ec2d77-6d55-443d-8457-8b68388e648d.jpg)